### ISSUE TITLE
boinccmd --get_host_info does not show GPUs

### ISSUE THREAD

**Issue by smoe** [BOINC/boinc#3210]
This former GPU mining rig still has a few non-functioning riser cards as it seems, so for now I only got 4 of the 6 graphic cards to be recognized by lspci, clinfo and ... how I interpreted it ... also by the boinc client, which when started directly gives the following to stdout:
```
05-Jul-2019 18:42:34 [---] Data directory: /var/lib/boinc-client
05-Jul-2019 18:42:34 [---] OpenCL: AMD/ATI GPU 0: Radeon RX 580 Series (driver v                                                                                    ersion 2841.17, device version OpenCL 1.2 AMD-APP (2841.17), 4059MB, 4059MB avai                                                                                    lable, 6175 GFLOPS peak)
05-Jul-2019 18:42:34 [---] OpenCL: AMD/ATI GPU 1: Radeon RX 580 Series (driver v                                                                                    ersion 2841.17, device version OpenCL 1.2 AMD-APP (2841.17), 4059MB, 4059MB avai                                                                                    lable, 6175 GFLOPS peak)
05-Jul-2019 18:42:34 [---] OpenCL: AMD/ATI GPU 2: Radeon RX 580 Series (driver v                                                                                    ersion 2841.17, device version OpenCL 1.2 AMD-APP (2841.17), 4059MB, 4059MB avai                                                                                    lable, 6175 GFLOPS peak)
05-Jul-2019 18:42:34 [---] OpenCL: AMD/ATI GPU 3: Radeon RX 580 Series (driver v                                                                                    ersion 2841.17, device version OpenCL 1.2 AMD-APP (2841.17), 4059MB, 4059MB avai                                                                                    lable, 6175 GFLOPS peak)
05-Jul-2019 18:42:34 [---] OpenCL: AMD/ATI GPU 4 (ignored by config): Radeon RX                                                                                     580 Series (POLARIS10, DRM 3.31.0, 4.18.0-25-generic, LLVM 8.0.0) (driver versio                                                                                    n 19.0.2, device version OpenCL 1.1 Mesa 19.0.2, 13107MB, 13107MB available, 385                                                                                    9 GFLOPS peak)
05-Jul-2019 18:42:34 [---] OpenCL: AMD/ATI GPU 5 (ignored by config): Radeon RX                                                                                     580 Series (POLARIS10, DRM 3.31.0, 4.18.0-25-generic, LLVM 8.0.0) (driver versio                                                                                    n 19.0.2, device version OpenCL 1.1 Mesa 19.0.2, 13107MB, 13107MB available, 385                                                                                    9 GFLOPS peak)
05-Jul-2019 18:42:34 [---] OpenCL: AMD/ATI GPU 6 (ignored by config): Radeon RX                                                                                     580 Series (POLARIS10, DRM 3.31.0, 4.18.0-25-generic, LLVM 8.0.0) (driver versio                                                                                    n 19.0.2, device version OpenCL 1.1 Mesa 19.0.2, 13107MB, 13107MB available, 385                                                                                    9 GFLOPS peak)
05-Jul-2019 18:42:34 [---] OpenCL: AMD/ATI GPU 7 (ignored by config): Radeon RX                                                                                     580 Series (POLARIS10, DRM 3.31.0, 4.18.0-25-generic, LLVM 8.0.0) (driver versio                                                                                    n 19.0.2, device version OpenCL 1.1 Mesa 19.0.2, 13107MB, 13107MB available, 385                                                                                    9 GFLOPS peak)
```
Running Einstein, I get only three tasks running. This seems somewhat plausible since there are only 2 active cores of the processor - but then it should be only two tasks, not 3. And if three are possible, then so would be four, right?

I thought that the "--get_host_info" would help with some clarification, but this does not mention GPUs at all:
```
$ boinccmd --get_host_info
  timezone: 7200
  domain name: rig1
  IP addr: 127.0.1.1
  #CPUS: 2
  CPU vendor: GenuineIntel
  CPU model: Intel(R) Pentium(R) CPU G4400 @ 3.30GHz [Family 6 Model 94 Stepping 3]
  CPU FP OPS: 4410268571.310606
  CPU int OPS: 140225282777.495148
  CPU mem BW: 1000000000.000000
  OS name: Linux Ubuntu
  OS version: Ubuntu 18.04.2 LTS [4.18.0-25-generic|libc 2.27 (Ubuntu GLIBC 2.27-3ubuntu1)]
  mem size: 33614630912.000000
  cache size: 3145728.000000
  swap size: 2147479552.000000
  disk size: 983429038080.000000
  disk free: 921980497920.000000
```

**Comment by smoe** [comment]
Side-note: The app_config.xml is how to adjust the demands for CPU and GPU per task. An example is given here https://einsteinathome.org/de/content/former-gpu-mining-rig-now-running-boinc#comment-172028 .

**Comment by AenBleidd** [comment]
Should be fixed in #3231
